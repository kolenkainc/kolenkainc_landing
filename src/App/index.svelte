<script>
    import Layout from '@Components/Layout';
    import ImageExample from '@Components/ImageExample';
    import HelloWorld from '@Components/HelloWorld';
    import Title from '@Components/Title';
</script>

<style src="./style.scss">

</style>

<Layout>
    <div class="container">
        <Title/>
    </div>
</Layout>
