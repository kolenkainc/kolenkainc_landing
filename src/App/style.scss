@font-face {
    font-family: 'Academy';
    src: url('../Assets/fonts/Academy.svg') format('svg');
    src: url('../Assets/fonts/Academy.ttf') format('truetype');
    font-weight: normal;
    font-style: normal;
}

.container {
    font-family: Academy;
    background-color: #fafafa;
    display: flex;
    flex-direction: column;
    flex: 1 1 100%;
    justify-content: flex-start;
    align-items: center;
    min-height: 100vh;
}
