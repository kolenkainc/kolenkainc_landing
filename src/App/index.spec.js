import App from './index.svelte'
import { render } from '@testing-library/svelte'

describe('App', () => {
    it('should render the app', () => {
        const { container, getAllByText } = render(App);

        expect(container.getElementsByClassName('farewell').length).toBe(1);
        expect(container.getElementsByClassName('title').length).toBe(1);
        expect(container.getElementsByClassName('image-content').length).toBe(1);
        expect(container.getElementsByClassName('smile-image').length).toBe(1);
        expect(container.getElementsByClassName('description').length).toBe(1);
        expect(container.getElementsByClassName('team').length).toBe(1);

        expect(getAllByText('Чертополох').length).toBe(1);
        expect(getAllByText('Команда Kolenka Inc.').length).toBe(1);

        expect(container).toMatchSnapshot();
    })
})
