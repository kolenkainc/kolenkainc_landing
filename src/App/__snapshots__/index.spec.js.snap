// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the app 1`] = `
<body>
  <div>
    <div
      class="layout"
    >
      <div
        class="container"
      >
        <div
          class="farewell"
        >
          <div
            class="child"
          >
            <div
              class="title"
            >
              Чертополох
            </div>
             
            <div
              class="image-content"
            >
              <div
                class="smile-image"
              />
            </div>
             
            <div
              class="description"
            >
              <p>
                В прошлом году мы попробовали идею о том, 
            как можно недорого и удобно для всех (зрителей / контролеров 
            / организаторов) продавать билеты на концерты.
              </p>
               
              <p>
                Чтобы развиваться дальше нужно развивать сервис. 
            К сожалению, сейчас мы не видим 
            большой выгоды в его развитии — поэтому временно закрываем проект.
              </p>
               
              <p>
                Возможно мы будем периодически открывать продажи билетов 
            на какие-то специальные мероприятия.
              </p>
               
              <p>
                Но как итог этого полугодия — 
            хочется сердечно поблагодарить всех, 
            кто не боялся покупать электронные билеты 
            и писал в техподдержку. Нашей команде правда очень жаль 
            расставаться со всеми вами!
              </p>
               
              <div
                class="team"
              >
                <p>
                  Команда Kolenka Inc.
                </p>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
